Plugins/Plugin.Test/Plugin.Test.nuspec

```diff
@@ -22,7 +22,7 @@
             <dependency id="NUnit" version="3.13.1" />
             <dependency id="Moq" version="4.16.1" />
             <dependency id="NLog" version="4.7.9" />
-            <dependency id="YamlDotNet" version="10.0.0" />
+            <dependency id="YamlDotNet" version="10.1.0" />
```

Comment:
Somehow this change isn't being picked up by the version checker?!

`  Error Message:
     Package YamlDotNet is version 10.1.0 in C:\projects\rdmp\Reusable\Tests\ReusableCodeTests\bin\Release\net5.0\../../../../../../Tests.Common/Tests.Common.csproj but version 10.0.0 in C:\projects\rdmp\Reusable\Tests\ReusableCodeTests\bin\Release\net5.0\../../../../../../Plugins/Plugin.Test/Plugin.Test.nuspec
  String lengths are both 6. Strings differ at index 3.
  Expected: "10.1.0"
  But was:  "10.0.0"
  --------------^
  Stack Trace:
     at ReusableCodeTests.NuspecIsCorrectTests.TestDependencyCorrect(String csproj, String nuspec, String packagesMarkdown) in C:\projects\rdmp\Reusable\Tests\ReusableCodeTests\NuspecIsCorrectTests.cs:line 86
`

Bizarre bug somewhere - I'll park this until next week unless anyone has any inspiration before then.